test/unit/Reflection/ReflectionFunctionTest.php

```diff
@@ -81,13 +81,22 @@ public function testIsUserDefined()
         $this->assertTrue($function->isUserDefined());
     }
 
-    public function testStaticCreation()
+    public function testStaticCreationFromName()
     {
         require_once(__DIR__ . '/../Fixture/Functions.php');
         $reflection = ReflectionFunction::createFromName('BetterReflectionTest\Fixture\myFunction');
         $this->assertSame('myFunction', $reflection->getShortName());
     }
 
+    public function testStaticCreationFromClosure()
```

Comment:
Question: what happens with two closures on the same line? Can you please try adding a test case with that scenario (I know it will fail, but at least we'd have a documented limitation)